uo_sys_framework/Core/cfgXEH/Killed/CAManBaseKilled.sqf

```diff
@@ -5,7 +5,7 @@ EXEC_CHECK(ALL);
 params ["_unit"];
 
 LOG("Killed_Event called");
-[QGVAR(KilledEvent), _unit] call CBA_fnc_serverEvent;
+//[QGVAR(KilledEvent), _unit] call CBA_fnc_serverEvent;
```

Comment:
These events are for use later in wave respawn/player dead handling of counts and unit vars by the server

Reply:
So it is intended that the killed functions on the server are called twice?

Reply:
the XEH fires local to unit

Reply:
GVAR(EventRespawnedHandle) = addMissionEventHandler ["EntityRespawned", {_this call FUNC(EventRespawned);}];
GVAR(EventKilledHandle) = addMissionEventHandler ["EntityKilled", {_this call FUNC(EventKilled);}];
GVAR(EventDisconnectHandle) = addMissionEventHandler ["HandleDisconnect", {_this call FUNC(EventDisconnect);}];

Reply:
These are also there calling the exact same function. Don't really know.

Reply:
If this breaks or could break something, it needs to be addressed before this PR goes forward.

Reply:
Do those functions exit on player/client? Could have set up two different systems for AI and Players. Should be all on XEH with a serverevent and isPlayer check

Reply:
Events were just blank if I remember correctly, so shouldnt have any effective change. They were for the future when server needed info on players dying/respawning

Reply:
> 
> 
> Events were just blank if I remember correctly, so shouldnt have any effective change. They were for the future when server needed info on players dying/respawning

So sacher's changes to these events is good to go ahead with?